templates/home.html

```diff
@@ -17,56 +17,91 @@
     <div id="home-page" class="page">
         <div class="heading-center"></div>
         <h1 class="home-title">PERDIEM</h1>
-        <h2 class="home-description">CROWDSOURCED INVESTING IN MUSIC</h2>
+        <h2 class="home-description">THE NEXT GENERATION OF MUSIC</h2>
         {% if user.is_authenticated %}
             <p class="welcome-text">Welcome {% firstof user.get_full_name user.get_username %}!</p>
         {% else %}
-            <p class="home-text"><a data-open="login-modal">LOGIN</a></p>
-            <p class="welcome-text">...or <a href="{% url 'register' %}">create an account</a>.</p>
+            <div class="home-boxes">
+            <button> <a class="button text-center box-margin" href={% url 'home' %}#how>I'm an artist</a></button>
```

Comment:
You should be able to take out the `<button>` and `</button>` tags here because `<a class="button">` and `</a>` will draw a button.